gulpfile.js

```diff
@@ -5,6 +5,27 @@ const electron = require('electron-connect').server.create({
 });
 const tape = require('gulp-tape');
 const tapColorize = require('tap-colorize');
+const ts = require("gulp-typescript");
+const tsProject = ts.createProject("tsconfig.json");
+const paths = {
+    files: [
+        'app/**/*',
+        '!app/**/*.ts'
+    ]
+};
+
+gulp.task("copy-non-ts", function () {
```

Comment:
We don't need to do this here, since we plan to move the regular `tsc` to output to `src`.

Reply:
We can manually remove already outputted files by doing `git clean -xf`.

Reply:
But, doesn't ```tsc``` just compile .ts to .js . This is for other files like css and html and images.

Reply:
Ah, ok makes sense!